gems/sorbet/lib/hidden-definition-finder.rb

```diff
@@ -114,7 +114,7 @@ def write_constants
     )
     File.write(SOURCE_CONSTANTS, io.read)
     io.close
-    raise "Your source can't be read by Sorbet.\nYou can try `rm sorbet/config; find . -type f | xargs -L 1 -t srb tc --error-white-list 1000` and hopefully the last file it is processing before it dies is the culprit.\nIf not, maybe the errors in this file will help: #{SOURCE_CONSTANTS_ERR}" if File.read(SOURCE_CONSTANTS).empty?
+    raise "Your source can't be read by Sorbet.\nYou can try `find . -type f | xargs -L 1 -t srb tc --no-config --error-white-list 1000` and hopefully the last file it is processing before it dies is the culprit.\nIf not, maybe the errors in this file will help: #{SOURCE_CONSTANTS_ERR}" if File.read(SOURCE_CONSTANTS).empty?
 
     puts "Printing #{TMP_RBI}'s symbol table into #{RBI_CONSTANTS}"
     # Change dir to deal with you having a sorbet/config in your cwd
```

Comment:
@ptarjan What do you think about taking @XrXr's  suggestion about removing this `chdir` and adding the `--no-config` flag?

Reply:
sure, PRs welcome!